src/System.Drawing.Common/src/System/Drawing/Font.Unix.cs

```diff
@@ -224,27 +84,20 @@ public static Font FromHfont(IntPtr hfont)
                 newStyle |= FontStyle.Bold;
             }
 
-            if (lf.lfHeight < 0)
-            {
-                newSize = lf.lfHeight * -1;
-            }
-            else
-            {
-                newSize = lf.lfHeight;
-            }
-
-            return (new Font(newObject, lf.lfFaceName, newStyle, newSize));
+            return new Font(newObject, lf.lfFaceName, newStyle, Math.Abs(lf.lfHeight));
```

Comment:
This would appear to change behavior in the case where lfHeight == int.MinValue.  Previously it would just end up passing in int.MinValue.  Now Math.Abs will throw an OverflowException.  Is it possible that lfHeight could be int.MinValue here?

Reply:
Fixed